### PR TITLE
created template for merge request onto main

### PR DESCRIPTION
**Pull request by stigsberg**
Creating template for merge

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+**Description of your changes:**
```
Comment (swedchef) [inline .github/pull_request_template.md:1]: I perceive this line as a heading not a listing and therefore suggest you remove the colon :)
```suggestion
**Description of your changes**
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+Category:
+
+* [ ] New Feature
+* [ ] Bugfix
+* [ ] Documentation fixed
+
+Checklist:
+
+* [ ] I have bumped the chart version
```
Comment (swedchef) [inline .github/pull_request_template.md:11]: There might be versioning for the application and the chart - perhaps we would like to have the commit'er check both?
```suggestion
* [ ] I have bumped all relevant version numbers
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+**Description of your changes:**
+
+Category:
+
+* [ ] New Feature
+* [ ] Bugfix
+* [ ] Documentation fixed
```
Comment (swedchef) [inline .github/pull_request_template.md:7]: Just a wording thing.. :-P
```suggestion
* [ ] Documentation update
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+Checklist:
+
+* [ ] I have bumped the chart version
+* [ ] I have commented my code, particularly in hard-to-understand areas
+* [ ] I have documented my changes if this is needed
+* [ ] I have described my changes in the "description of your changes" field
+* [ ] I have tested this change on a running cluster 
+* [ ] I have created the necessary tests in the chart, if they are possible/necessary
```
Comment (swedchef) [inline .github/pull_request_template.md:16]: I am uncertain if misunderstood this line? What chart?
```suggestion
* [ ] I have created/corrected automated tests related to the changes, if possible/necessary
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+* [ ] New and existing unit tests pass locally with my changes
```
Comment (swedchef) [inline .github/pull_request_template.md]: Locally?

**Review by swedchef — APPROVED** [review]
Nothing breaking - but a few suggestions.
I don't know how exactly it is implemented, so I can't check the functionality - only the text :)

File: .github/pull_request_template.md (hunk 5 above)
Comment (stigsberg) [reply .github/pull_request_template.md]: Yes, think it is okay that people have tested pylint before PR.

**Pull request merged**